### PR TITLE
databases/week3/meal_sharing.sql

### PR DESCRIPTION
**Pull request by saloumeh-67**

### CODE REVIEW

File: databases/week3/meal_sharing.sql [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+FROM meal 
+WHERE 2021-09-01 OR 2021-09-03;
+--Get only specific number of meals fx return only 5 meals
+SELECT * from meal WHERE id > 3;
+--Get the meals that have good reviews
+SELECT * from review WHERE stars > 4;
+--Get reservations for a specific meal sorted by created_date
+SELECT meal.id, reservation.meal_id, meal.title, 
+reservation.number_of_guests, reservation.created_date FROM meal, reservation
+WHERE meal.title LIKE "carbonerra" and meal.id=reservation.meal_id
```
Comment (Elia-Git) [inline databases/week3/meal_sharing.sql:101]: Why we need to select this specific meal(WHERE meal. title LIKE "carbonara")? You don't need it. You have to sort out all meals according to their create_date
hint
```
SELECT meal.Title, reservation.meal_Id, SUM(reservation.Number_of_guests), reservation.Created_date AS Res_created_date
FROM reservation
   JOIN meal ON (meal.id=reservation.meal_id)
GROUP BY reservation.created_date,meal.Title
ORDER BY reservation.created_date;
```

File: databases/week3/meal_sharing.sql [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+SELECT * from meal WHERE title LIKE '%pasta%';
+--Get meals that has been created between two dates
+UPDATE meal
+SET created_date = '2021-09-3'
+WHERE id = 3;
+SELECT created_date
+FROM meal 
+WHERE 2021-09-01 OR 2021-09-03;
+--Get only specific number of meals fx return only 5 meals
+SELECT * from meal WHERE id > 3;
```
Comment (Elia-Git) [inline databases/week3/meal_sharing.sql:95]: what are you aiming to get here?
hint:use limit

File: databases/week3/meal_sharing.sql (hunk 1 above)
Comment (saloumeh-67) [reply databases/week3/meal_sharing.sql:101]: it gives me back this error "  SELECT list is not in GROUP BY clause and contains nonaggregated column 'meal_sharing.reservation.meal_id' which is not functionally dependent on columns in GROUP BY clause; this is incompatible with sql_mode=only_full_group_by".?!!

Comment (Elia-Git) [reply databases/week3/meal_sharing.sql:101]: Should give out like this
image

**Review by Elia-Git — APPROVED** [review]
Pls ship it if commented gives desired results !

**Pull request merged**